docs/preview/features/show-dotnet-templates-in-visualstudio.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Show .NET project templates in Visual Studio"
+layout: default
+---
+
+# Show .NET project templates in Visual Studio
+
+The available .NET project templates in this repository are all available via the command line.
+For example:
+
+```shell
+PM > dotnet new --install Arcus.Templates.WebApi
+```
+
+Which will install the template on your machine so it can be used for creating new projects:
+
+```shell
+> dotnet new arcus-webapi --name Arcus.Demo.WebAPI
+```
+
+But these .NET project templates can also be shown up in Visual Studio by following these steps:
+
+1. Go to **Tools > Options** in Visual Studio
+2. Navigate to **Preview Features**
+3. Check the box **Show all .NET Core templates in the New project dialog**
+
+![Show all .NET Core templates in the New project dialog via Tools > Options > Preview Features](img/show-dotnet-templates-in-vs.png)
+
```

Comment:
Couldn't yet show how the result is because we don't have any templates released yet with the logo.

Reply:
Let's split the PRs then maybe?

Reply:
Yeah, ok, I'll remove this and the image from the PR.